src/components/AggregateList.tsx

```diff
@@ -108,6 +111,7 @@ export default class AggregateList extends React.Component<AggregateListProps, A
   }
 
   toggleFormatting() {
+    // Status of the Remove/Restore Formatting button--determines whether CSS styles are applied to the data
```

Comment:
The comments are great, but comments like these that describe the complete function (rather than specific lines) or that describe the complete class should be written above the function or class declaration. If you have time, can you please convert the comments to go above each function? And for classes, start the comments with 

```
/*
 * Description goes here.
 */
class Something() {
...
```

Reply:
If an example is helpful, [this](https://github.com/NYPL-Simplified/circulation-web/blob/master/src/components/CirculationEventsDownloadForm.tsx) is probably a useful format/level of detail